rust/ql/test/library-tests/dataflow/models/models.expected

```diff
@@ -555,23 +277,21 @@ subpaths
 | main.rs:202:19:202:19 | s | main.rs:201:14:201:14 | ... | main.rs:201:17:201:42 | if ... {...} else {...} | main.rs:202:13:202:23 | apply(...) |
 | main.rs:202:19:202:19 | s | main.rs:201:14:201:14 | ... | main.rs:201:17:201:42 | if ... {...} else {...} | main.rs:202:13:202:23 | apply(...) |
 testFailures
+| main.rs:42:28:42:46 | //... | Missing result: hasValueFlow=2 |
+| main.rs:57:37:57:55 | //... | Missing result: hasValueFlow=3 |
+| main.rs:74:30:74:48 | //... | Missing result: hasValueFlow=4 |
+| main.rs:89:51:89:69 | //... | Missing result: hasValueFlow=5 |
+| main.rs:109:40:109:58 | //... | Missing result: hasValueFlow=6 |
+| main.rs:129:29:129:47 | //... | Missing result: hasValueFlow=7 |
+| main.rs:236:51:236:70 | //... | Missing result: hasValueFlow=12 |
+| main.rs:244:51:244:70 | //... | Missing result: hasValueFlow=13 |
+| main.rs:254:47:254:66 | //... | Missing result: hasValueFlow=14 |
+| main.rs:261:15:261:34 | //... | Missing result: hasValueFlow=15 |
```

Comment:
Do we know why these fail?

Reply:
yes, the diff on the expected file looks much better now!